ext/standard/image.c

```diff
@@ -27,7 +27,7 @@
 #endif
 #include "php_image.h"
 #ifdef PHP_WIN32
-#include "win32/php_stdint.h"
+# include <stdint.h>
 #endif
```

Comment:
Can we drop the inclusion of stdint.h altogether here?

Reply:
Yes, this and the two cases below can be dropped, i.e.
````.diff
 ext/standard/image.c | 3 ---
 main/SAPI.h          | 3 ---
 main/php_variables.c | 3 ---
 3 files changed, 9 deletions(-)

diff --git a/ext/standard/image.c b/ext/standard/image.c
index c55af95c54..e967aaeee0 100644
--- a/ext/standard/image.c
+++ b/ext/standard/image.c
@@ -26,9 +26,6 @@
 #include <unistd.h>
 #endif
 #include "php_image.h"
-#ifdef PHP_WIN32
-# include <stdint.h>
-#endif
 
 #if HAVE_ZLIB && !defined(COMPILE_DL_ZLIB)
 #include "zlib.h"
diff --git a/main/SAPI.h b/main/SAPI.h
index e6ef24720e..d1ef8a40b0 100644
--- a/main/SAPI.h
+++ b/main/SAPI.h
@@ -22,9 +22,6 @@
 #include "zend_API.h"
 #include "zend_llist.h"
 #include "zend_operators.h"
-#ifdef PHP_WIN32
-# include <stdint.h>
-#endif
 #include <sys/stat.h>
 
 #define SAPI_OPTION_NO_CHDIR 1
diff --git a/main/php_variables.c b/main/php_variables.c
index 81f8a7b9ec..b41286312d 100644
--- a/main/php_variables.c
+++ b/main/php_variables.c
@@ -25,9 +25,6 @@
 #include "php_content_types.h"
 #include "SAPI.h"
 #include "zend_globals.h"
-#ifdef PHP_WIN32
-# include <inttypes.h>
-#endif
 
 /* for systems that need to override reading of environment variables */
 void _php_import_environment_variables(zval *array_ptr);
````